lib/vagrant-openshift/action/generate_template.rb

```diff
@@ -40,7 +40,7 @@ def call(env)
           box_info = box_info_data[os][stage]
           box_info[:instance_name] = @options[:name].nil? ? 'openshift_origin_' + inst_ts : @options[:name]
           box_info[:os] = os
-          box_info[:vagrant_guest] = [:centos7, :rhel7].include?(os) ? :redhat : os
+          box_info[:vagrant_guest] = [:centos7, :rhel7, :rhelatomic].include?(os) ? :redhat : os
```

Comment:
So the current naming scheme is:

We only support 1 version of fedora at a time.  And fedora moves to fast to put a version number on it and change it all the time.  For rhel and centos we have historically supported more than 1 version at a time.  And they don't change very often so we put the version number in the name.